helmfile/charts/networkpolicy-common/values.yaml

```
@@ -0,0 +1,40 @@
+fullnameOverride: ""
+nameOverride: ""
+
+global:
+  apiserver:
+    ips:
+      - "0.0.0.0/0"
+    port: 6443
+  ingress:
+    ips:
+      - "0.0.0.0/0"
+  nodes:
+    ips:
+      - "0.0.0.0/0"
+  objectStorage:
+    ips:
+      - "0.0.0.0/0"
+    ports:
+      - 443
+
+  monitoringSelector:
```

I think this is a bit more accurate
```suggestion
  prometheusSelector:
```

---

But I really like this idea, it could be adopted in more places.

---

Yeah, a refactor run will be needed after we get all these merged. I think it will be the easiest way to make sure we reuse parts like this.